README.md

```diff
@@ -82,6 +82,8 @@ OPTIONS:
 
 ## Building
 
+`scripts/build`
```

Comment:
Unfortunately this doesn't work for me, it complains that I didn't clone the repo over downloading the zip.